server/routes/terms.js

```diff
@@ -27,6 +27,24 @@ router.get('/', async (req, res) => {
     })
 })
 
+router.get('/counts', async (req, res) => {
+  var query = {
+    text: 'SELECT DISTINCT term, COUNT (term) FROM entry GROUP BY term'
```

Comment:
I think this needs a clause specifying that `entry.action=2`. Entries can have an `action`, or status, of 1-4 (potential, accepted, rejected, reported), and we only want to show terms that have an entry that's accepted.

Reply:
New query:
```
SELECT DISTINCT LOWER(term) AS term, COUNT(term)
FROM entry
WHERE action=2
GROUP BY term
ORDER BY count DESC, term ASC
```